package.json

```diff
@@ -24,13 +24,13 @@
     "@ionic-native/deeplinks": "^3.4.4",
     "@ionic-native/in-app-browser": "^3.4.4",
     "@ionic-native/toast": "^3.4.4",
-    "@ionic/storage": "1.1.7",
+    "@ionic/storage": "^2.0.1",
     "angular2-jwt": "^0.1.27",
     "ionic-angular": "2.2.0",
     "ionic-native": "2.8.1",
     "ionic2-input-mask": "^0.18.6",
     "ionicons": "3.0.0",
-    "rxjs": "5.0.0-beta.12",
+    "rxjs": "^5.3.0",
```

Comment:
I'll make a note on slack to inform everyone they need to run npm update before running the code.